dbt/scripts/export_qc_town_close_reports.py

```diff
@@ -0,0 +1,228 @@
+# Export town close QC reports to Excel files.
+#
+# Run `python scripts/export_qc_town_close_reports.py --help` for details.
+import argparse
+import contextlib
+import datetime
+import io
+import json
+import os
+import sys
+
+from dbt.cli.main import dbtRunner
+
+# Add the parent directory of `scripts` to the module search path
+# so that we can import from other modules in the `scripts` directory
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Hmmmm, yeah this is pretty wonky. Why not move some of the export functionality into a `utils/` dir/module. Then `dbt/scripts/export_models.py` can be a simple wrapper for the `utils/` code, and both `export_models.py` and `export_qc_town_close_reports.py` can share the utils code.

Reply:
That makes sense! If it's cool with you, I'll do that in a fast-follow PR so that you can review it with confidence that no other parts of the code have changed during the file moving process.